taxcalc/tests/test_pufcsv.py

```diff
@@ -239,3 +239,42 @@ def test_mtr(tests_path, puf_path):  # pylint: disable=redefined-outer-name
         msg += '---            and rerun test.                ---\n'
         msg += '-------------------------------------------------\n'
         raise ValueError(msg)
+
+
+@pytest.mark.requires_pufcsv
+def test_credit_reforms(puf_path):  # pylint: disable=redefined-outer-name
```

Comment:
@martinholmer, do you envision these tests catching errors that the /comparison results would not, or is this primarily a safety guard for when comparison/reform_results.py is not run?